common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -527,10 +557,9 @@ def _render_student_view_for_items(self, context, display_items, fragment, view=
                 # The item url format can be defined in the template context like so:
                 # context['item_url'] = '/my/item/path/{usage_key}/whatever'
                 iteminfo['href'] = context.get('item_url', '').format(usage_key=usage_id)
-            if is_user_authenticated and render_items:
-                if item.location.block_type == 'vertical':
-                    if completion_service:
-                        iteminfo['complete'] = completion_service.vertical_is_complete(item)
+            if is_user_authenticated:
```

Comment:
Why do we no longer have a check for `render_items` here?

Reply:
I introduced the `render_items` conditional when adding the sequence module UI navigation to Studio. In that case, there's no need for completion and bookmark checks, so I hid them behind the conditional. Now we need the completion info for the Unit, regardless of whether it's rendered.